### ISSUE TITLE
OpenFermion logo

### ISSUE THREAD

**Issue by babbush** [quantumlib/OpenFermion#55]
We'd like to come up with a logo for OpenFermion. Among other things, if we get a good logo, the odds of eventually making OpenFermion schwag increases dramatically. We can probably get some professional assistance in actually drawing up the logo but it would help to crowd source some concepts for the design. Perhaps the logo should be a stylized way of writing OpenFermion with some symbol. For instance, look at the [PySCF GitHub](https://github.com/sunqm/pyscf). It would be nice to replace the big "OpenFermion" letters with something more flashy.

Any logo should at least convey that the package has something to do with electrons. Ideally, the logo would also suggest some connection to quantum computing. One idea is that we could draw up a Bloch sphere (like [this one](https://upload.wikimedia.org/wikipedia/commons/f/f4/Bloch_Sphere.svg)) where the circumference (dotted in the one I linked to) is one of those Bohr model electron orbital/rings that everyone associates with electrons (like those in the PySCF logo). Any other ideas?

**Comment by kanavsetia** [comment]
Spherical Harmonics are really cool too! I was looking at this picture
https://www.flickr.com/photos/oskay/4498896878/
and trying to come up with ideas. I'll let you know, if I come up with something.

**Comment by babbush** [comment]
I like those too. Maybe the first "O" in OpenFermion could be the 3s orbital or something. Too bad none of the other letters have enough symmetry to be orbitals!

**Comment by kanavsetia** [comment]
That's exactly what I was thinking too! There are plenty of options for "O", but just couldn't find an "F"
Also, if you just take one of the orbitals, then it's kinda hard to guess that someone was going for spherical harmonics. So, I was thinking something like this
https://user-images.githubusercontent.com/2586352/31698882-c263c26a-b38d-11e7-8612-68a9956842e1.png
"F" made out of spherical harmonics. And top row could be made out of just s orbitals for "O"s. 
But, this wouldn't be very clear if the logo is printed small (which generally is the case).

**Comment by babbush** [comment]
Yeah I am also concerned that if we make the letters out of multiple atomic orbitals then it will be too small to see the orbitals. Let's try to imagine something that could be printed in reasonable sized letters on the breast of a t-shirt or something.

**Comment by JohanWinther** [comment]
![openfermion](https://user-images.githubusercontent.com/28558941/32943967-2ce53746-cb8e-11e7-8636-1f1dcc3d685a.png)
Here is one concept, it is subtle but I think it gets too cluttered when I tried to add a spin arrow to any of the O:s in the logo.

**Comment by babbush** [comment]
I like it! is it possible to make the "e" in Fermion even more stylized? That does give a good nod to the electrons. I do think we need a subtle nod to quantum computing, probably via the O. What if the O were a little thinner, more round, and had a coordinate axis inside so that it looked like a Bloch sphere? I could see that being too cluttered. Another option would be to put either the first O, the last n, or both in bra's / kets. For instance, <O|penFermion or <O|penFermio|n> or OpenFermio|n>?

**Comment by babbush** [comment]
Especially if the O is a "0", this sort of makes sense. Because people often write things like H|n> = E_n |n>, so we are often after the |0> state.

**Comment by babbush** [comment]
I think less is more though. Perhaps just <O|penF_e_rmion would be best.

**Comment by JohanWinther** [comment]
![image](https://user-images.githubusercontent.com/28558941/33343341-3262f050-d485-11e7-8843-e7ade38a6b1c.png)

By going [here](https://www.codecogs.com/eqnedit.php?latex=\dpi{300}&space;\huge&space;\hat{\textbf{O}}\textbf{pen}\text{F}e\text{rmio}|\text{n}\rangle) you can see for yourself how different configurations look. I am not actually a designer (merely self-taught) so my opinions should probably be taken with a grain of salt. I like the way the O looks with the hat although, which is a shame since I don't really know if there is an O-operator in QC.

It would however also be nice if there could be a logo version with just the O (like the short version of the Amazon logo for example). Something that would make you instantly recognize it. But on the other hand this isn't exactly the next blockbuster brand.
I like where this is going so let's see some more brainstorming and novel ideas!

**Comment by ncrubin** [comment]
<img width="868" alt="openfermion_logo" src="https://user-images.githubusercontent.com/7128746/33388745-9befcab0-d4e5-11e7-84b8-4d234cf4391d.png">

Alludes to a 2-Fermion diagram (minus the arrows) and also a molecule?  The diagram in the upper right can certainly be cleaned up/more and stylized.

**Comment by babbush** [comment]
I like both of these. While I don't think we've found "the ultimate" OpenFermion logo yet, I am in favor of using one of these on the front page sooner rather than later (we can always change it later on). @JohanWinther did you make the first logo using latex as well? If so, perhaps you can share the latex? I'd like to try one that is sort of a combination of the two examples you gave. Perhaps in a couple days we should have a vote among the OpenFermion developers to determine the first logo.

**Comment by JohanWinther** [comment]
The logo was created using Illustrator, but I've created a [ShareLatex project](https://www.sharelatex.com/read/tzrrrcpwgvnh) which you can check out.

**Comment by schuylerfried** [comment]
@ncrubin's design is pretty sleek

**Comment by babbush** [comment]
I like @ncrubin's Feynman diagram but I despise his font. So what if we wrote OpenFermion like [this](https://www.codecogs.com/eqnedit.php?latex=%5Cdpi%7B300%7D&space;%5Chuge&space;%5Chat%7B%5Ctextbf%7BO%7D%7D%5Ctextbf%7Bpen%7D%5Ctext%7BF%7De%5Ctext%7Brmio%7D%7C%5Ctext%7Bn%7D%5Crangle) but where the "e" is some sort of orange/yellow color (I feel electrons are yellow but it needs to be dark enough to be easy to read). Furthermore, instead of the \rangle on the n in the ket, we put Nick's Feynman diagram (to close the ket). Also, it should be aligned as Nick has it so that Open / the Feynman diagram fits above "Fermion". Do others share my vision here!?

**Comment by ncrubin** [comment]
I think serif fonts for logos should be avoided.  But that's my personal preference.   Font for the logo I suggested is certainly negotiable.  Mostly a proposal for using a non-serif font with non-latex characters + diagram.  I can work on a couple of versions.   I think the |n>...<  thing might look a little cluttered but I can make a rough sketch of it.

**Comment by babbush** [comment]
I'm totally cool losing the serifs. I think @JohanWinther's font in the first example is nice (and that doesn't have serifs). But I do like the idea of having an italics e that is orange in Fermion, a hat on O in Open, and n in Open inside a ket and the Feynman diagram closing that ket.

**Comment by babbush** [comment]
@ncrubin a month back you showed me some designs at IBM conference and we talked about small modifications that would make everyone happy. Any progress on those? I'm serious about trying to get some schwag for March meeting but we'll need to decide on a logo first!

**Comment by ncrubin** [comment]
![openfermion](https://user-images.githubusercontent.com/7128746/34739176-54484cd0-f530-11e7-8976-aa4284daf2d3.png)

Integrating some of the ideas from above: Feynman diagram is cleaned up a little, diagram is now part of $\vert n \rangle$, orange for fermion (hat-tip @babbush), different font.  I think the diagram is still a little heavy on the ket lines but that can be changed when this is mocked up in a design software suite.

**Comment by ncrubin** [comment]
![openfermion_2](https://user-images.githubusercontent.com/7128746/34739773-36b43ccc-f532-11e7-981e-1b27098e0086.png)
slightly darker border on the circles in the diagram.

**Comment by babbush** [comment]
I love it.

**Comment by ncrubin** [comment]
![openfermion_3](https://user-images.githubusercontent.com/7128746/34740858-a87a9cea-f535-11e7-8e7e-973d9390a765.png)
adjusted F-e spacing and n-ket spacing.  O-F vertical and n-bra vertical are now aligned

**Comment by jarrodmcc** [comment]
Just wanted to leave a comment saying I think it looks really good!

**Comment by babbush** [comment]
I actually think I like the font that Open was written in, in the second-to-last one.

**Comment by babbush** [comment]
Logo is now on repo front page. I'll update with more professional version once we get it touched up.

**Comment by babbush** [comment]
It might also be cool to see a version of exactly this with them both on the same line. That might look better on the GitHub page.
